Example/README.md

```diff
@@ -1,7 +1,32 @@
 # Example
 
+Check out [stripe-samples](https://github.com/stripe-samples) for examples to:
+
+- [Accept a card payment](https://github.com/stripe-samples/accept-a-card-payment) (PaymentIntents API)
+- [Accept a card payment](https://github.com/stripe-samples/card-payment-charges-api) (Charges API)
+- [Save a card without payment](https://github.com/stripe-samples/mobile-saving-card-without-payment) (SetupIntents API)
+
 There are 3 example apps included in the repository:
 
-- [**UI Examples** Example/UI Examples/README.md ](/Example/UI%20Examples/README.md)
-- [**Basic Integration** Example/Basic Integration/README.md](/Example/Basic%20Integration/README.md)
-- [**Custom Integration** Example/Custom Integration/README.md](/Example/Custom%20Integration/README.md)
+## [**Basic Integration**](/Example/Basic%20Integration/README.md)
+<p align="center">
+<img src="https://raw.githubusercontent.com/stripe/stripe-ios/a87e2fb12ce1ba6b45a075ee22e0da5072a54279/standard-integration.gif" width="240" alt="STPPaymentCardTextField" align="center">
+</p>
+
+This example app demonstrates how to build a payment flow using our prebuilt UI component integration. 
+ 
+ 
+## [**Custom Integration** Example/Custom Integration/README.md](/Example/Custom%20Integration/README.md)
```

Comment:
There's probably a better name for Custom Integration.  It's more like 'APM Examples'.  

"Non-card Payment Examples"?
"Other Payment Methods"?

Reply:
I like "Non-Card Payment Examples"

Reply:
Okay with keeping "Custom Integration" for now as well